mcp_can.cpp:285:15: error: 'CAN_666KBPS' was not declared in this scope

case sensitive error 

mcp_can.cpp:285:15: error: 'CAN_666KBPS' was not declared in this scope
         case (CAN_666KBPS):
               ^
Error compiling.

in file mcp_can_dfs.h  line no 395
#define CAN_666kbps  17
change to 
#define CAN_666KBPS  17

It's work.

--- a/mcp_can_dfs.h
+++ b/mcp_can_dfs.h
@@ -392,7 +392,7 @@
 #define CAN_200KBPS  14
 #define CAN_250KBPS  15
 #define CAN_500KBPS  16
-#define CAN_666kbps  17
+#define CAN_666KBPS  17
 #define CAN_1000KBPS 18
 
 #define CAN_OK                  (0)

